cdisc_rules_engine/check_operators/sql/base_sql_operator.py

```diff
@@ -182,6 +182,15 @@ def _column_sql(
         # TODO: Throwing this temporarily, so we can determine which errors
         # are actually postgres errors and which are just rules which run on
         # optional variables without checking
+
+        if column == "dataset_name":
```

Comment:
Can you make this a constant somewhere? Also it looks like you've split a comment from the thing it's describing. Can you do a similar thing for `DOMAIN` as well